<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Organizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    font-family: Arial, sans-serif;
    font-size: 14px;
}
QPushButton {
    padding: 6px 10px;
    border: 1px solid #ccc;
    border-radius: 4px;
    background-color: #f5f5f5;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #e5e5e5;
}
QPushButton:disabled {
    background-color: #f8f8f8;
    color: #999;
}
QCheckBox {
    spacing: 6px;
    font-size: 14px;
}
QCheckBox::indicator {
    width: 16px;
    height: 16px;
}
QLabel {
    font-size: 14px;
    color: #333;
}
QDateEdit {
    padding: 6px 8px;
    border: 1px solid #ccc;
    border-radius: 4px;
    background-color: white;
    font-size: 14px;
}
QDateEdit:focus {
    border-color: #4CAF50;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="folderLabel">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="folder_entry">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    padding: 6px 8px;
    border: 1px solid #ccc;
    border-radius: 4px;
    background-color: white;
    font-size: 14px;
}</string>
        </property>
        <property name="text">
         <string>No folder selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browse_btn">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="recursive_cb">
        <property name="text">
         <string>Include current folder and subfolders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="skip_confirm_cb">
        <property name="text">
         <string>delete confirmation</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="scan_btn">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 4px;
    font-weight: bold;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #45a049;
}</string>
        </property>
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="folderNavLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="folder_info">
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #666;</string>
        </property>
        <property name="text">
         <string>No folders found</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="prev_folder_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>◀ Folder</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_folder_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #da190b;
}
QPushButton:disabled {
    background-color: #ccc;
    color: #666;
}</string>
        </property>
        <property name="text">
         <string>Delete Folder</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_folder_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Folder ▶</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Right</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="media_display">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: #f8f9fa;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
    color: #666;
    font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>No media selected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mediaNavLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="media_info">
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #666;</string>
        </property>
        <property name="text">
         <string>No media files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="prev_media_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>◀ Media</string>
        </property>
        <property name="shortcut">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="play_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #2196F3;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #1976D2;
}
QPushButton:disabled {
    background-color: #ccc;
    color: #666;
}</string>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_media_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>105</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #da190b;
}
QPushButton:disabled {
    background-color: #ccc;
    color: #666;
}</string>
        </property>
        <property name="text">
         <string>Delete Media</string>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_media_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Media ▶</string>
        </property>
        <property name="shortcut">
         <string>Right</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="status">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #666; font-size: 13px;</string>
      </property>
      <property name="text">
       <string>Ready</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
